mod/wet4/views/default/event_calendar/repeat_form_element.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+$event = $vars['event'];
+$fd = $vars['form_data'];
+echo '<div class="event-calendar-repeat-section">';
+if ($fd['repeats'] == 'yes') {
+	echo elgg_view('input/checkbox', array('name' => 'repeats', 'value' => 'yes', 'checked' => 'checked'));
+} else {
+	echo elgg_view('input/checkbox', array('name' => 'repeats', 'value' => 'yes'));
+}
+echo elgg_echo('event_calendar:repeat_interval_label').' ';
+echo elgg_view('input/dropdown', array('name' => 'repeat_interval','style'=>'width: auto; display:inline-block;', 'value' => $fd['repeat_interval'], 'options_values' => array('1' => '1', '2' => '2', '3' => '3', '4' => '4', '5' => '5', '6' => '6', '7' => '7', '8' => '8')));
+echo ' '.elgg_echo('event_calendar:repeat_weeks');
+echo ' '.elgg_echo('event_calendar:on_these_days');
+
+echo '<div class="event-calendar-repeating-wrapper">';
+
+$days = array('monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday', 'sunday');
+
+foreach ($days as $day) {

```

Comment:
I change a little bit the text and add a show and hide option depending if the checkbox is checked